### PR TITLE
Fce 319 update other endpoints

### PR DESCRIPTION
**Pull request by roznawsk**

### CODE REVIEW

File: .circleci/continue_config.yml [hunk 1]
```diff
@@ -80,7 +80,7 @@ jobs:
 
     resource_class: large
     docker:
-      - image: membraneframeworklabs/docker_membrane:v2.3.0
+      - image: membraneframeworklabs/docker_membrane:v2.4.0
```
Comment (roznawsk) [inline .circleci/continue_config.yml:83]: Elixir v1.16

File: file/lib/file_source_endpoint.ex
Comment (roznawsk) [inline file/lib/file_source_endpoint.ex:1]: Some changes caused by api change in ogg plugin

File: file/test/fixtures/video.h264
Comment (roznawsk) [inline file/test/fixtures/video.h264:1]: made the video shorter (faster tests)

File: file/lib/file_source_endpoint.ex [hunk 2]
```diff
@@ -142,36 +142,14 @@ defmodule Membrane.RTC.Engine.Endpoint.File do
     actions =
```
Comment (Karolk99) [inline file/lib/file_source_endpoint.ex:142]: please add infromation in module docs that this module now is only compatible with engine that has only one track

**Pull request merged**